.github/workflows/test.yml

```diff
@@ -198,7 +198,9 @@ jobs:
 
       - name: Start
         working-directory: ./
-        run: docker-compose -f docker-compose.gha.yml up --build -d
+        run: |
+          docker compose -f docker-compose.gha.yml build
+          docker compose -f docker-compose.gha.yml up -d
```

Comment:
Why do we split this up by the way? Isn't this the same as before?
https://docs.docker.com/reference/cli/docker/compose/up/

Reply:
Have unified